doc/man7/EVP_PKEY-ML-DSA.pod

```diff
@@ -69,6 +79,127 @@ respective key type of B<ML-DSA-44>, B<ML-DSA-65> or B<ML-DSA-87>.
 
 =back
 
+=head2 Provider configuration parameters
+
+See the description of the B<-provparam> option in L<openssl(1)> to learn
+how to set provider configuration parameters in the command line tools.
+See L<OSSL_PROVIDER_add_conf_parameter(3)> to learn how to set provider
+configuration options programmatically.
+
+=over 4
+
+=item C<ml-dsa.retain_seed> (B<OSSL_PKEY_PARAM_ML_DSA_RETAIN_SEED>) <UTF8 string>
+
+When set to a string representing a false boolean value (see
+L<OSSL_PROVIDER_conf_get_bool(3)>), the seed will not be retained after key
+generation or key import from a seed value.
+If the resulting key is then written to a PKCS#8 object, it will contain
+only the FIPS 204 C<sk> key.
+
+=item C<ml-dsa.prefer_seed> (B<OSSL_PKEY_PARAM_ML_DSA_PREFER_SEED>) <UTF8 string>
+
+When decoding PKCS#8 objects that contain both a seed and the FIPS 204 C<sk>
+private key, the seed is by default used to regenerate the key, and the
+companion private key is ignored.
+When this configuration parameter is set to a string representing a false
+boolean value (see L<OSSL_PROVIDER_conf_get_bool(3)>), the seed is ignored
+(neither used to regenerate the key, nor retained), and the companion key is
+used instead.
+
+=item C<ml-dsa.input_formats> (B<OSSL_PKEY_PARAM_ML_DSA_INPUT_FORMATS>) <UTF8 string>
+
+List of enabled private key input formats when parsing PKCS#8 objects.
+List elements are separated by commas, spaces or tabs.
+The list of enabled formats can be specified in the configuration file, as seen
+in the L</EXAMPLES> section below, or the via the B<-provparam> command-line
+option (see also L<OSSL_PROVIDER_add_conf_parameter(3)>).
+
+Values specified on the command-line override any configuration file settings.
+By default all the supported formats are enabled.
+The supported formats are:
+
+=over 4
+
+=item C<seed-priv>:
+
+This format represents B<PKCS#8> objects in which both the FIPS 204 32-byte
+B<ξ> seed and the secret key B<sk> are present in the private key as part of
+the DER encoding of the ASN.1 sequence:
+
+ PrivateKey ::= SEQUENCE {
+    seed OCTET STRING OPTIONAL,
+    expandedKey [1] IMPLICIT OCTET STRING OPTIONAL }
+   (WITH COMPONENTS {..., seed  PRESENT } |
+    WITH COMPONENTS {..., expandedKey  PRESENT })
+
+If the C<seed-priv> format is not included in the list, this format will not be
+recognised on input.
+
+=item C<seed-only>:
+
+This format represents B<PKCS#8> objects in which only the 32-byte FIPS 204
+B<ξ> seed is present in the above sequence.
+If the C<seed-only> format is not included in the list, this format will not be
+recognised on input.
+
+=item C<priv-only>:
+
+This format represents B<PKCS#8> objects in which only the FIPS 204
+decapsulation key B<sk> is present in the above sequence.
+If the C<priv-only> format is not included in the list, this format will not be
+recognised on input.
+
+=item C<oqs-kpair>:
```

Comment:
why kpair?

Reply:
Well it is a key pair, (private + public). Bikeshedding the name is guess OK, if you feel there's an obviously better name.

Reply:
pair or keypair